docs/source/onedal/algorithms/covariance/covariance.rst

```diff
@@ -37,6 +37,52 @@ Given a dataset :math:`X = \{ x_1, \ldots, x_n \}` with :math:`n` feature vector
 the means is a :math:`1 \times p` matrix, the covariance and the correlation matrices are :math:`p \times p` square matrices.
 The means, the covariance, and the correlation are computed with the following formulas:
 
+.. list-table::
+   :widths: 20 50
+   :header-rows: 1
+   :align: left
+
+   * - Statistic
+     - Definition
+   * - Means
+     - :math:`M = (m_{1}, \ldots , m_{p})`, where :math:`m_{j}=\frac{1}{n}\sum _{i}{x}_{ij}`
+   * - Covariance matrix (sample)
+     - :math:`Cov = (v_{ij})`, where :math:`v_{ij}=\frac{1}{n-1}\sum_{k=1}^{n}(x_{ki}-m_{i})(x_{kj}-m{j})`, :math:`i=\overline{1,p}`, :math:`j=\overline{1,p}`
+   * - Covariance matrix (maximum likelihood)
+     - :math:`Cov' = (v'_{ij})`, where :math:`v'_{ij}=\frac{1}{n}\sum_{k=1}^{n}(x_{ki}-m_{i})(x_{kj}-m{j})`, :math:`i=\overline{1,p}`, :math:`j=\overline{1,p}`
```

Comment:
I think this equation is not needed here because we always compute unbiased sample covariance matrix as far as I understand.

Reply:
No, we support biased and unbiased since https://github.com/oneapi-src/oneDAL/pull/2581